src/sugar/orderbook.ts

```diff
@@ -81,7 +84,7 @@ async function getOrderbook(
   const buy: BookOffer[] = []
   const sell: BookOffer[] = []
   orders.forEach((order) => {
-    if (order.Flags === OfferCreateTransactionFlags.tfSell) {
+    if (order.Flags === SELL_OFFER_FLAGS) {
```

Comment:
This shouldn't be `===` - that doesn't account for the possibility that multiple flags are turned on